### PR TITLE
chore/refactor-all-packages-json

### PR DESCRIPTION
**Pull request by PatrykKuniczak**
> `*` Please fill in the required items.

## Priority*

- [ ] High: This PR needs to be merged first for other tasks.
- [x] Middle: This PR should be merged quickly to prevent conflicts due to common changes. (default)
- [ ] Low: This PR does not affect other tasks, so it can be merged later.

## Purpose of the PR*
I want to do it in more `turbo` friendly way, and i've done it as simply as possible for now.

## Changes*
I've split `clean` command and remove redundancy from `dev`  

I've used `pnpx` for avoid cases when users run `pnpm clean:turbo` and then want to run `pnpm clean:node_modules` or `clean`

By this move i want to avoid a bunch of issues about new error(users reports) 😆 

I've also removed tasks from `dependsOn` due to lack of opportunities to define a order, in `package.json` i can do it freely.
Maybe in the future we'll find a better solution for that. 

And also don't pay attension to `cross-env` because it works for now, and i'll try to refactor it, to looks and works better 😄 

## How to check the feature
Run new and old commands, and check if it works

### CODE REVIEW

File: packages/i18n/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@extension/i18n",
   "version": "0.3.1",
-  "description": "chrome extension internationalization",
+  "description": "chrome extension - internationalization",
```
Comment (Jonghakseo) [inline packages/i18n/package.json:4]: 👍🏼

File: package.json [hunk 2]
```diff
@@ -7,19 +7,23 @@
...
-    "clean:turbo": "turbo clean && turbo daemon stop && rimraf dist && rimraf .turbo",
-    "clean": "pnpm clean:turbo && pnpm clean:node_modules",
+    "clean-root:bundle": "pnpx rimraf dist && pnpx rimraf dist-zip",
+    "clean-root:node_modules": "pnpx turbo daemon stop && pnpx rimraf node_modules",
+    "clean-root:turbo": "pnpx turbo daemon stop && pnpx rimraf .turbo",
+    "clean-root": "pnpm clean-root:bundle && pnpm clean-root:node_modules && pnpm clean-root:turbo",
+    "clean:node_modules": "pnpx turbo clean:node_modules && pnpm clean-root:node_modules",
+    "clean:turbo": "pnpx turbo clean:turbo && pnpm clean-root:turbo",
+    "clean:bundle": "pnpx turbo clean:bundle && pnpm clean-root:bundle",
+    "clean": "pnpx turbo clean && pnpm clean-root",
```
Comment (Jonghakseo) [inline package.json:19]: OK... There's too many clean scripts lol

I think in many case we need this clean scripts just for clean entire projects stuff. So I suggest leaving just the clean-root scripts.

```suggestion
    "clean:bundle": "pnpx rimraf dist && pnpx rimraf dist-zip",
    "clean:node_modules": "pnpx turbo daemon stop && pnpx rimraf node_modules",
    "clean:turbo": "pnpx turbo daemon stop && pnpx rimraf .turbo",
    "clean": "pnpm clean-root:bundle && pnpm clean-root:node_modules && pnpm clean-root:turbo",
```

**Review by Jonghakseo — COMMENTED** [review]
👍🏼

File: chrome-extension/tsconfig.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": "@extension/tsconfig/app",
   "compilerOptions": {
-    "baseUrl": "./",
+    "baseUrl": ".",
```
Comment (Jonghakseo) [inline chrome-extension/tsconfig.json:4]: Was this change necessary?

Comment (PatrykKuniczak) [reply chrome-extension/tsconfig.json:4]: No, but i want to do it, to have this config like all others

File: package.json (hunk 2 above)
Comment (PatrykKuniczak) [reply package.json]: That suggestion isn't fully good, because of the mistake in `clean` but it's good change to make, i've done it.

I need to run turbo for the rest `clean:` scripts, check it and feel free to request more changes if needs :)

**Review by Jonghakseo — APPROVED** [review]
LGTM
